### ISSUE TITLE
Check dependencies build command failed

### ISSUE THREAD

**Issue by marns** [yukiarrr/ios-build-action#21]
I'm getting a kind of nondescript error:
```
[22:12:17]: ▸ ** ARCHIVE FAILED **
160
[22:12:17]: ▸ The following build commands failed:
161
[22:12:17]: ▸ 	Check dependencies
```

Any ideas how to debug / fix?

Step:
```
      - name: Build IPA
        uses: yukiarrr/ios-build-action@v1.1.1
        with:
          project-path: iOS/iOS/Unity-iPhone.xcodeproj
          p12-base64: ${{ secrets.P12_BASE64 }}
          mobileprovision-base64: ${{ secrets.MOBILEPROVISION_BASE64 }}
          code-signing-identity: ${{ secrets.CODE_SIGNING_IDENTITY }}
          team-id: ${{ secrets.TEAM_ID }}
          disable-targets: "UnityFramework,Unity-iPhone"
```

Job output:

```
[22:12:15]: ▸ Building Unity-iPhone/UnityFramework [Release]
158
[22:12:15]: ▸ Check Dependencies
159
[22:12:17]: ▸ ** ARCHIVE FAILED **
160
[22:12:17]: ▸ The following build commands failed:
161
[22:12:17]: ▸ 	Check dependencies
162
[22:12:17]: ▸ (1 failure)
163
    The Legacy Build System will be removed in a future release. You can configure the selected build system and this deprecation message in File > Project Settings.
164
▸ Cleaning Unity-iPhone/UnityFramework [Release]
165
▸ Check Dependencies
166
▸ Cleaning Unity-iPhone/Unity-iPhone [Release]
167
▸ Check Dependencies
168
▸ Clean Succeeded
169
    The Legacy Build System will be removed in a future release. You can configure the selected build system and this deprecation message in File > Project Settings.
170
▸ Building Unity-iPhone/UnityFramework [Release]
171
▸ Check Dependencies
172
** ARCHIVE FAILED **
173

174

175
The following build commands failed:
176
	Check dependencies
177
(1 failure)
178
[22:12:17]: Exit status: 65
179
[22:12:17]: 
180
[22:12:17]: Maybe the error shown is caused by using the wrong version of Xcode
181
[22:12:17]: Found multiple versions of Xcode in '/Applications/'
182
[22:12:17]: Make sure you selected the right version for your project
183
[22:12:17]: This build process was executed using '/Applications/Xcode_12.app'
184
[22:12:17]: If you want to update your Xcode path, either
185
[22:12:17]: 
186
[22:12:17]: - Specify the Xcode version in your Fastfile
187
[22:12:17]: ▸ xcversion(version: "8.1") # Selects Xcode 8.1.0
188
[22:12:17]: 
189
[22:12:17]: - Specify an absolute path to your Xcode installation in your Fastfile
190
[22:12:17]: ▸ xcode_select "/Applications/Xcode8.app"
191
[22:12:17]: 
192
[22:12:17]: - Manually update the path using
193
[22:12:17]: ▸ sudo xcode-select -s /Applications/Xcode.app
194
[22:12:17]: 
195

196
+---------------+----------------------------+
197
|             Build environment              |
198
+---------------+----------------------------+
199
| xcode_path    | /Applications/Xcode_12.app |
200
| gym_version   | 2.160.0                    |
201
| export_method | app-store                  |
202
| sdk           | iPhoneOS14.0.sdk           |
203
+---------------+----------------------------+
```

**Comment by marns** [comment]
Found the rest of the log file- better error is:

> [35mCode Signing Error: UnityFramework does not support provisioning profiles. UnityFramework does not support provisioning profiles, but provisioning profile [myapp] has been manually specified. Set the provisioning profile value to "Automatic" in the build settings editor.

Tried removing `UnityFramework` from `disable-targets`, and I get the same error. I see a similar issue under discussion here: https://github.com/microsoft/azure-pipelines-tasks/issues/12263
But it looks like you all are building signed Unity apps successfully.

Commenting out `disable-targets` (trying to test automatic signing), I get:

> Code Signing Error: UnityFramework has conflicting provisioning settings. UnityFramework is automatically signed, but provisioning profile [myapp] has been manually specified. Set the provisioning profile value to "Automatic" in the build settings editor, or switch to manual signing in the Signing & Capabilities editor.

So I'm not sure how to either test auto-signing for the whole app, or build/sign UnityFramework correctly with a signed app

**Comment by marns** [comment]
I can see this is already being addressed in https://github.com/yukiarrr/ios-build-action/issues/18

I confirmed that using @gpiffaretti's ios-build-action@master passed this step, so I think this one can be closed.

I will open a new issue for the next error I'm running into: https://github.com/yukiarrr/ios-build-action/issues/22

**Comment by yukiarrr** [comment]
Including the PR that resolved this issue, v1.2.0 has been released.
Thanks for your report!
